src/js/components/SkipLink/SkipLink.js

```diff
@@ -1,10 +1,10 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
 
 import { Anchor } from '../Anchor';
 import { Box } from '../Box';
 
-export const SkipLink = ({ id, label, ...rest }) => (
+export const SkipLink = forwardRef(({ id, label, ...rest }, ref) => (
   <Box margin="small">
```

Comment:
I'm wondering if `SkipLinks` should control all of the layout and use `gap` and `margin` and leave `SkipLink` to be just the `Anchor`?

Reply:
You read my mind. 
I actually have code in place that does just that, but then I thought maybe we should keep the Box and SkipLink will have a theme to style this Box container and the user will be able to control margin, padding, or any other Box property for an individual SkipLink. 
Two of the approaches sounded like they could work for me, so I said I should leave it as is and see which need arises :)
If you have a strong preference I'm good either way.

Reply:
Since it's a "Link", I think it should just be an "Anchor". I don't think we need to do too much styling for it, since it's designed more for users with sight issues.

Reply:
I think SkipLink should just be Anchor. Since this is for users with sight issues, I don't think we need to invest a lot for styling it.

Reply:
Ok, I'll clean it up.

Reply:
Done.